assets/scripts/users/user_flags.js

```diff
@@ -0,0 +1,19 @@
+export function receiveUserFlags (flags) {
```

Comment:
Should we combine this file with the `flag_utils` module?

Reply:
Oh sure! `flag_utils` module is under `app` so should I move the `user_flags.test.js` under `app` as well?

Reply:
Yep!